draft-ietf-quic-transport.md

```diff
@@ -2224,8 +2237,9 @@ PATH_RESPONSE frames.
 ### Successful Path Validation
 
 Path validation succeeds when a PATH_RESPONSE frame is received that contains
-the data that was sent in a previous PATH_CHALLENGE frame. This validates the
-path on which the PATH_CHALLENGE was sent.
+the data that was sent in a previous PATH_CHALLENGE frame.  A PATH_RESPONSE
+frame received on any network path validates the path on which the
+PATH_CHALLENGE was sent.
```

Comment:
I guess it actually only validates one direction of that path but I guess that nit picking...